test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -57,11 +57,11 @@
 
         (testing "should attempt a partial migration if there are migrations missing"
           (clear-db-for-testing!)
-          ;; We are using migration 13 here because it is isolated enough to be able
+          ;; We are using migration 28 here because it is isolated enough to be able
```

Comment:
This is modified because the report-id migration relies on latest-reports, so that comment there is no longer valid, in that this new migration relies on 13 having taken place. The migration chosen here really just needs to be independent from the others. I am going to make a new test for this migration though.